lib/binary_serialization.dart

```diff
@@ -435,6 +435,7 @@ class BinaryReader {
         TypedefInfo info = new TypedefInfo.internal();
         _readBasicInfo(info);
         info.type = source.readString();
+        return null;
```

Comment:
oops, yay linter found a problem! ... it should be `return info;` though.

Reply:
Eek. Done.